certif/tests/acceptance/session-candidates-test.js

```diff
@@ -266,4 +248,13 @@ module('Acceptance | Session Candidates', function(hooks) {
 
   });
 
+  async function _fillFormWithCorrectData() {
+    await fillIn('[data-test-id="panel-candidate__lastName__add-staging"] > div > input', validName);
+    await fillIn('[data-test-id="panel-candidate__firstName__add-staging"] > div > input', validFirstName);
+    await fillIn('[data-test-id="panel-candidate__birthCity__add-staging"] > div > input', validCity);
+    await fillIn('[data-test-id="panel-candidate__birthProvinceCode__add-staging"] > div > input', validBirthCode);
+    await fillIn('[data-test-id="panel-candidate__birthCountry__add-staging"] > div > input', validCountry);
+    await fillIn('[data-test-id="panel-candidate__birthdate__add-staging"] > div > input', validBirthdate);
+  }
```

Comment:
Si on change le dom, ça sera plus simple pour adapter cette fonction : 
Mais par contre c'est peut être moins compréhensible ? Des avis ? 
```suggestion
  async function _fillFormWithCorrectData() {
    const firstPartSelector = "[data-test-id="panel-candidate__";
    const lastPartSelector = "__add-staging"] > div > input"
    await fillIn(`${firstPartSelector}lastName${lastPartSelector}`, validName);
    await fillIn(`${firstPartSelector}firstName${lastPartSelector}`, validFirstName);
    await fillIn(`${firstPartSelector}birthCity${lastPartSelector}`, validCity);
    await fillIn(`${firstPartSelector}birthProvinceCode${lastPartSelector}`, validBirthCode);
    await fillIn(`${firstPartSelector}birthCountry${lastPartSelector}`, validCountry);
    await fillIn(`${firstPartSelector}birthdate${lastPartSelector}`, validBirthdate);
  }
```

Reply:
J'ai plutôt scindé en 2 fonctions. Est-ce que ca te va @Anne-Gaelle-S ?